app/experimenter/experiments/api/v5/mutations.py

```diff
@@ -65,8 +66,20 @@ def mutate(cls, root, info, input: ExperimentInput):
         return handle_with_serializer(cls, serializer)
 
 
+class TriggerCheckKintoPushQueueNow(graphene.Mutation):
+    message = ObjectField()
+
+    @classmethod
+    def mutate(cls, root, info):
+        nimbus_check_kinto_push_queue()
+        return cls(message="Triggered nimbus_check_kinto_push_queue task")
```

Comment:
We didn't have an issue for adding this to the backend. But, [the Figma mock](https://www.figma.com/proto/5SLa07XqQb0ttB1ay1gami/Nimbus-Console-UI?node-id=218363%3A30350&scaling=min-zoom) has a note to call the Remote Settings API to immediately create the new record. That seemed a bit dicey to do literally, so I punted and just made an attempt to kick off the Celery task that leads to that result.

![Capture](https://user-images.githubusercontent.com/21687/114634440-5ec5e180-9c77-11eb-8ae9-af9e2771e2c8.PNG)

Reply:
Oh oh.  Yeah okay so, I wasn't thinking of it like that.  If we want to do that we'd have to move some things around and change some logic.  The way it's setup now is

- Owner requests review
- Reviewer approves
- UI now shows above, but experiment is not yet in RS
- Next scheduled invocation of push task kicks in 0-5 minutes later
- Now it's available in RS to review

If we want to set it up so that when it moves to APPROVED we call the task immediately, we'd have to do a bunch of refactoring.  Like probably a lot.  So I would hazard that we stick with the scheduled approach for now and if we want to do it more immediately we'll have to scope and plan that separately.  cc @AnaMedinac

Reply:
If we keep the scheduled approach I'd probably recommend a note (tooltip?) in this container to explain the delay. Could be pretty confusing otherwise since you might not realize what's happening -- is it broken? do I need to talk to someone? why does it sometimes show up immediately and sometimes I have to wait 5 minutes?

Reply:
So, this mutation should be taken out?

Reply:
I'm just going to remove the mutation - we can follow up with an enhancement, possibly as part of [EXP-1200](https://jira.mozilla.com/browse/EXP-1200), since responsiveness of the background tasks is related here too.

Reply:
https://jira.mozilla.com/browse/EXP-1201